maintainers/maintainer-list.nix

```diff
@@ -2457,6 +2457,12 @@
     githubId = 40633781;
     name = "Sergei S.";
   };
+  diogox = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add diogox`` and sort it alphabetically.

Reply:
Done :raised_hands: